test/unit/base.test.js

```diff
@@ -37,6 +38,22 @@ describe('CONVERT FUNCTION TESTS ', function() {
         expect(conversionResult.output[0].data).to.have.property('info');
         expect(conversionResult.output[0].data).to.have.property('item');
 
+        done();
+      });
+    });
+    it('[Github: 102]- Should generate collection description with only given fields in info object ' +
+      specPath2, function(done) {
+      Converter.convert({ type: 'file', data: specPath2 }, { schemaFaker: true }, (err, conversionResult) => {
+        let description = conversionResult.output[0].data.info.description;
+        expect(err).to.be.null;
+        expect(conversionResult.result).to.equal(true);
+        expect(conversionResult.output.length).to.equal(1);
+        expect(conversionResult.output[0].type).to.equal('collection');
+        expect(conversionResult.output[0].data).to.have.property('info');
+        expect(conversionResult.output[0].data).to.have.property('item');
+        // eslint-disable-next-line max-len
```

Comment:
We disable the eslint rules only when necessary and have no other alternative. In this particular case, we can split the test across two lines without altering any behaviour of it. This rule insists on code readability.